src/app/public/modules/dropdown/dropdown.component.ts

```diff
@@ -87,12 +114,27 @@ export class SkyDropdownComponent implements OnInit, OnDestroy {
     return this._label || this.getString('skyux_dropdown_context_menu_default_label');
   }
 
+  /**
+   * Provides an observable to send commands to the dropdown. The commands should respect
+   * [the `SkyDropdownMessage` type](https://developer.blackbaud.com/skyux/components/dropdown#skydropdownmessage-type).
```

Comment:
Types that exist on the same page can be wrapped in double square brackets. `[[SkyDropdownMessage]]`